resqpy/fault.py

```diff
@@ -20,7 +20,7 @@
 import resqpy.olio.uuid as bu
 import resqpy.olio.write_hdf5 as rwh5
 import resqpy.olio.xml_et as rqet
-import resqpy.organize as rqo
+import resqpy.organize.organize_old as rqo
```

Comment:
@cflynn3 is this intentional? Doesn't look like a change that is ready to be accepted, but it has already been merged into the integration branch.